packages/core/src/lib/implementation/persist.unit.test.ts

```diff
@@ -41,31 +41,6 @@ describe('persistReport', () => {
     );
   });
 
-  it('should create a report in json format by default', async () => {
```

Comment:
Why has this been removed? It describes a valid use case from the core perspective - for each format the cases are tested here.

Reply:
the logic got changed and now this is not true